components/api/gen/schema.bal

```diff
@@ -95,10 +99,19 @@ public type OrgListResponse record {
     OrgListResponseAtom[] data;
 };
 
+public type OrgListAtom record {
```

Comment:
Any reason to have `Atom` as a part of the record type? Applicable for `ImagesListAtom` as well.

Reply:
Atom is used for denoting a single record/element in a response